includes/alerts.inc.php

```diff
@@ -37,7 +37,7 @@ function GenSQL($rule) {
         return false;
     }
     //Pretty-print rule to dissect easier
-    $pretty = array('*'  => '*', '('  => ' ( ', ')'  => ' ) ', '/'  => '/', '&&' => ' && ', '||' => ' || ', 'DATE_SUB ( NOW (  )' => 'DATE_SUB( NOW()');
+    $pretty = array('*'  => '*', '('  => ' ( ', ')'  => ' ) ', '/'  => '/ ', '&&' => ' && ', '||' => ' || ', 'DATE_SUB ( NOW (  )' => 'DATE_SUB( NOW()');
```

Comment:
Yeah, accidentally branched off my old PR :( Want me to submit fresh, or are you OK with my revert in the next commit?